<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValidationForm</class>
 <widget class="QWidget" name="ValidationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOperator">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>252</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>323</y>
     <width>16</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDataA">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>216</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDataB">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>286</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDataTotal">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>356</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="gpDataType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Type</string>
   </property>
   <widget class="QRadioButton" name="optDate">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dates</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Numbers</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gpOperationType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Operation Type</string>
   </property>
   <widget class="QRadioButton" name="optMinus">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Minus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optSum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sum</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="optMulti">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiply</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optDiv">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>60</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Divide</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCalculate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHelpDtFormat">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>225</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>dd/mm/yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHelpDayFormat">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>294</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>dd</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

